app/views/reports/show.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@report)%>_row").replaceWith("<%= j(render "reports/report", report: @report)%>");
```

Comment:
I think the `render` can be condensed [since it follows convention](https://api.rubyonrails.org/classes/ActionView/PartialRenderer.html).

```suggestion
$("#<%= dom_id(@report)%>_row").replaceWith("<%= j(render @report)%>");
```

What do you think?